src/main/resources/db/changelog/change/create-user-table.yaml

```diff
@@ -4,7 +4,7 @@ databaseChangeLog:
       author: kreker
       changes:
       - createTable:
-          tableName: user
+          tableName: users
```

Comment:
I think it's better to give table names in singular, as it was previously.
You can find more details [here](https://www.sqlshack.com/learn-sql-naming-conventions/), I think it's useful. But if in a project they already have table names in plural, you should follow project's style :)